src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -412,7 +412,7 @@ object IConvertible.ToType(Type type, IFormatProvider? provider)
         static byte IIncrementOperators<byte>.operator ++(byte value) => ++value;
 
         // /// <inheritdoc cref="IIncrementOperators{TSelf}.op_CheckedIncrement(TSelf)" />
```

Comment:
Out of curiosity: are the block comments excluded from the uncommenting pass purposely?

Reply:
Nope. That is a mistake and I'll fix it when I address the other relevant comments given above

Reply:
Fixed.